build.gradle

```diff
@@ -187,15 +180,7 @@ subprojects {
                 username geArtifactoryReader
                 password geArtifactoryReaderPassword
             }
-            url 'https://devcloud.swcoe.ge.com/artifactory/PREDIX'
-        }
-
-        maven {
-            credentials {
-                username geArtifactoryReader
-                password geArtifactoryReaderPassword
-            }
-            url 'https://devcloud.swcoe.ge.com/artifactory/PREDIX-SNAPSHOT'
+            url 'https://artifactory.build.ge.com/predix-virtual'
```

Comment:
This probably needs to be `MAAXA` and `MAAXA-SNAPSHOT`, not `predix-virtual`. It's used for resolving and deploying the child gradle projects (the actual functional ones).

Reply:
`MAAXA` and `MAAXA-SNAPSHOT` weren't required here to deploy. We need predix-virtual for audit sdk dependency.